diff --git a/libavfilter/af_astats.c b/libavfilter/af_astats.c
index 1ec56d72cc..bb9a23a017 100644
--- a/libavfilter/af_astats.c
+++ b/libavfilter/af_astats.c
@@ -496,7 +496,7 @@ static void set_metadata(AudioStatsContext *s, AVDictionary **metadata)
 
 #define UPDATE_STATS(planar, type, sample, normalizer_suffix, int_sample) \
     if ((s->measure_overall | s->measure_perchannel) & ~MEASURE_MINMAXPEAK) {                          \
-        UPDATE_STATS_##planar(type, update_stat(s, p, sample, sample normalizer_suffix, int_sample), s->is_float ? update_float_stat(s, p, sample) : s->is_double ? update_double_stat(s, p, sample) : NULL, );    \
+        UPDATE_STATS_##planar(type, update_stat(s, p, sample, sample normalizer_suffix, int_sample), s->is_float ? update_float_stat(s, p, sample) : s->is_double ? update_double_stat(s, p, sample) : (void)NULL, ); \
     } else {                                                                                           \
         UPDATE_STATS_##planar(type, update_minmax(s, p, sample), , p->nmin = p->min normalizer_suffix; p->nmax = p->max normalizer_suffix;); \
     }
